client/me/connected-application-item/index.jsx

```diff
@@ -120,116 +111,103 @@ module.exports = React.createClass( {
 
 		if ( message ) {
```

Comment:
Looks like we don't need to return anything if `message` isn't defined, in which case it would be nice if we returned early and nixed the indentation instead…

```js
if ( ! message ) {
	return;
}

return (
	<div>
		…
	</div>
);
```